app/utils/header-links.js

```diff
@@ -0,0 +1,177 @@
+export default [
+  {
+    name: 'Docs',
+    type: 'dropdown',
+    items: [
+      {
+        href: 'https://guides.emberjs.com',
+        name: 'Ember.js Guides',
+        type: 'link',
+      },
+      {
+        href: 'https://api.emberjs.com',
+        name: 'API Reference',
+        type: 'link',
+      },
+      {
+        href: 'https://cli.emberjs.com',
+        name: 'CLI Guides',
+        type: 'link',
+      },
+      {
+        type: 'divider',
+      },
+      {
+        href: 'https://emberjs.com/learn',
+        name: 'Learn Ember',
+        type: 'link',
+      },
+    ],
+  },
+  {
+    name: 'Releases',
+    type: 'dropdown',
+    items: [
+      {
+        href: 'https://emberjs.com/releases',
+        name: 'Overview',
+        type: 'link',
+      },
+      {
+        href: 'https://emberjs.com/releases/lts',
+        name: '→ LTS',
+        type: 'link',
+      },
+      {
+        href: 'https://emberjs.com/releases/release',
+        name: '→ Stable',
+        type: 'link',
+      },
+      {
+        href: 'https://emberjs.com/releases/beta',
+        name: '→ Beta',
+        type: 'link',
+      },
+      {
+        href: 'https://emberjs.com/releases/canary',
+        name: '→ Canary',
+        type: 'link',
+      },
+      {
+        type: 'divider',
+      },
+      {
+        href: 'https://emberjs.com/editions',
+        name: 'Editions',
+        type: 'link',
+      },
+      {
+        href: 'https://emberjs.com/browser-support',
+        name: 'Browser Support',
+        type: 'link',
+      },
+      {
+        href: 'https://deprecations.emberjs.com',
+        name: 'Deprecations',
+        type: 'link',
+      },
+      {
+        href: 'https://github.com/emberjs/rfc-tracking/issues',
+        name: 'RFC Tracking',
+        type: 'link',
+      },
+    ],
+  },
+  {
+    href: 'https://blog.emberjs.com',
+    name: 'Blog',
+    type: 'link',
+  },
+  {
+    name: 'Community',
+    type: 'dropdown',
+    items: [
+      {
+        href: 'https://emberjs.com/community',
+        name: 'The Ember Community',
+        type: 'link',
+      },
+      {
+        href: 'https://emberjs.com/guidelines',
+        name: 'Guidelines',
+        type: 'link',
+      },
+      {
+        href: 'https://help-wanted.emberjs.com/',
+        name: 'Help Wanted',
+        type: 'link',
+      },
+      {
+        type: 'divider',
+      },
+      {
+        href: 'https://emberjs.com/community/meetups',
+        name: 'Meetups',
+        type: 'link',
+      },
+      {
+        type: 'divider',
+      },
+      {
+        href: 'http://emberconf.com/',
+        name: 'Ember Conf',
+        type: 'link',
+      },
+    ],
+  },
+  {
+    name: 'About',
+    type: 'dropdown',
+    items: [
+      {
+        href: 'https://emberjs.com/teams',
```

Comment:
Note, in `ember-styleguide`, the value for `href` is `'https://emberjs.com/team'`. We'll want to make another pull request in `ember-styleguide` to fix the URL.